docs/devel/DEVELOPMENT.md

```diff
@@ -1,4 +1,4 @@
 ## Development
 
 The development guide has become part of the Contributing documentation
-and can be found [here](https://www.openpolicyagent.org/docs/edge/development/).
```

Comment:
I think we should link to the proper page on edge -- after all, it'll become the released version of the docs with the next release.

Alternatively, and that might be preferable, we could move this file back to its previous location -- so the URL path would just stay intact: https://github.com/open-policy-agent/opa/pull/4079/files#diff-84e02be4aa716ca7b59412d8488104a3caee1df1f59f51603792aee08d586462